sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormTrainingAsyncClientJavaDocCodeSnippets.java

```diff
@@ -54,16 +55,17 @@ public void beginTraining() {
         // BEGIN: com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.beginTraining#string-boolean
         String trainingFilesUrl = "{SAS-URL-of-your-container-in-blob-storage}";
         boolean useTrainingLabels = true;
-        formTrainingAsyncClient.beginTraining(trainingFilesUrl, useTrainingLabels).subscribe(
-            trainingPollingOperation -> {
+        formTrainingAsyncClient.beginTraining(trainingFilesUrl, useTrainingLabels)
+            .subscribe(trainingPollingOperation -> {
```

Comment:
like other change, change to flatMap()?

Reply:
We din't want two subscribes nested so changed for others. Not applicable here.